src/views/ObserveGames/ObserveGames.tsx

```diff
@@ -190,7 +191,7 @@ export class ObserveGames extends React.PureComponent<ObserveGamesProperties, an
                 </div>
             </div>
 
-            <GameList list={this.state.game_list} disableSort={true} emptyMessage={_("No games being played")} />
+            <GameList list={this.state.game_list} disableSort={false} emptyMessage={_("No games being played")} />
```

Comment:
The problem with enabling sorting here is that it doesn't actually sort the entire list, only the current page. Support for sorting needs to be added on the server side to support this.